// @project        The CERN Tape Archive (CTA)
// @copyright      Copyright(C) 2003-2021 CERN
// @license        This program is free software: you can redistribute it and/or modify
//                 it under the terms of the GNU General Public License as published by
//                 the Free Software Foundation, either version 3 of the License, or
//                 (at your option) any later version.
//
//                 This program is distributed in the hope that it will be useful,
//                 but WITHOUT ANY WARRANTY; without even the implied warranty of
//                 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
//                 GNU General Public License for more details.
//
//                 You should have received a copy of the GNU General Public License
//                 along with this program.  If not, see <http://www.gnu.org/licenses/>.
syntax = "proto2";
package castor.messages;

message ForkCleaner {
  // Description of the tape drive
  required string unitname = 1;
  required string logicallibrary = 2;
  required string devfilename = 3;
  required string libraryslot = 4;

  // Description of the tape associated with the drive
  required string vid = 5;

  // Description of the cleaner job
  required bool waitMediaInDrive = 6;
  required uint32 waitMediaInDriveTimeout = 7;
}
